src/api/svg.ts

```diff
@@ -0,0 +1,537 @@
+      y: options.y - element.svgAttributes.y,
+      borderColor: element.svgAttributes.stroke,
+      borderWidth: element.svgAttributes.strokeWidth,
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+      scale: element.svgAttributes.scale,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async image(element) {
+    page.drawImage(await page.doc.embedPng(element.svgAttributes.src), {
+      x: options.x + element.svgAttributes.x,
+      y: options.y - element.svgAttributes.y,
+      width: element.svgAttributes.width,
+      height: element.svgAttributes.height,
+      opacity: element.svgAttributes.fillOpacity,
+      xSkew: element.svgAttributes.skewX,
+      ySkew: element.svgAttributes.skewY,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async rect(element) {
+    if (!element.svgAttributes.fill && !element.svgAttributes.stroke) return;
+    page.drawRectangle({
+      x: options.x + element.svgAttributes.x,
+      y: options.y - element.svgAttributes.y,
+      width: element.svgAttributes.width,
+      height: element.svgAttributes.height,
+      borderColor: element.svgAttributes.stroke,
+      borderWidth: element.svgAttributes.strokeWidth,
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+      xSkew: element.svgAttributes.skewX,
+      ySkew: element.svgAttributes.skewY,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async ellipse(element) {
+    page.drawEllipse({
+      x: options.x + element.svgAttributes.x + element.svgAttributes.cx,
+      y: options.y - element.svgAttributes.y - element.svgAttributes.cy,
+      xScale: element.svgAttributes.rx,
+      yScale: element.svgAttributes.ry,
+      borderColor: element.svgAttributes.stroke,
+      borderWidth: element.svgAttributes.strokeWidth,
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+      rotate: element.svgAttributes.rotate,
+    });
+  },
+  async circle(element) {
+    page.drawCircle({
+      x: options.x + element.svgAttributes.x + element.svgAttributes.cx,
+      y: options.y - element.svgAttributes.y - element.svgAttributes.cy,
+      size: element.svgAttributes.r,
+      borderColor: element.svgAttributes.stroke,
+      borderWidth: element.svgAttributes.strokeWidth,
+      borderOpacity: element.svgAttributes.strokeOpacity,
+      borderLineCap: element.svgAttributes.strokeLineCap,
+      color: element.svgAttributes.fill,
+      opacity: element.svgAttributes.fillOpacity,
+    });
+  },
+});
+
+const transform = (
```

Comment:
The rotate and scale transformation are wrong. You cannot convert only x and y. What needs to change is actually the whole Converter object.